### PR TITLE
TMVCConfigKey.ServerName and Iocp.Utils

### PR DESCRIPTION
**Pull request by littleearth**
Hi, Ability to set different server name, and check for reason Iocp.Utils is missing?

### CODE REVIEW

File: sources/MVCFramework.Logger.pas [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+{ See the License for the specific language governing permissions and }
+{ limitations under the License. }
+{ }
+{ *************************************************************************** }
 
 unit MVCFramework.Logger;
 
 interface
 
+{$DEFINE MVC_LOGENABLED}
```
Comment (danieleteti) [inline sources/MVCFramework.Logger.pas]: This is a breaking change. If a user update the framework, then it loose the log with no notice. I prefer a more conservative update. Also, the logger is already configurable without change the core files. Can you leave only the "servername" related change? Thank you.

Comment (littleearth) [inline sources/MVCFramework.Logger.pas]: I have reverted, I cannot see how to configure option for the log, or is this just in reference to be able to modify MVCFramework.Logger.pas?

**Comment by danieleteti** [comment]
So now what contains this PR in terms of functionalities?

**Comment by littleearth** [comment]
TMVCConfigKey.ServerName - specify a different server name
and missing  Iocp.Utils

**Comment by danieleteti** [comment]
Added the changes about ServerName but this PR is too old now, so I cannot merge it without a lot of merging work. However your changes should be there.
Thank you

**Pull request closed without merging**